library/src/main/java/com/okta/auth/AuthorizationCallback.java

```diff
@@ -0,0 +1,16 @@
+package com.okta.auth;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.okta.openid.appauth.AuthorizationException;
+
+public interface AuthorizationCallback {
```

Comment:
onCancel is called when user enters wrong password or backs out of the custom tab.
onStatus is a update on the progress of authorization. Its up to the app to display for users or an be ignored.

Reply:
@FeiChen-okta I think we can use onError in order to pass cancelation and I don't think that is a good idea to send status updates if you can maybe show some use case where those statuses can be useful? It would be cool if we can reduce our callback to 2 methods `onSuccess` and `onError`

Reply:
Status is like a progress update. Good to have but not necessary so it can be removed. I would like to keep cancel and error separate since these are different events. Don't want to overload onError for cancel.